src/blocks/remote-data-container/components/panels/OverridesPanel.tsx

```diff
@@ -53,6 +53,7 @@ export function OverridesPanel( props: OverridesPanelProps ) {
 					key={ override.name }
 					label={ override.display_name || override.name }
 					onChange={ enabled => updateOverrides( override.name, enabled ) }
+					__nextHasNoMarginBottom
```

Comment:
I've been fixing all the deprecation warnings in the console, so this is part of that